resources/Feed-icon.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>

```

Comment:
We should probably rename this file to something like `rss.svg`, makes more sense imo.
Remember to update other places in code!